@here/harp-mapview/lib/FrustumIntersection.ts

```diff
@@ -142,19 +170,24 @@ export class FrustumIntersection {
 
                 const geoBox = getGeoBox(tilingScheme, childTileKey, offset);
 
+                // For tiles without elevation range source, default 0 will be propagated
+                // as min and max elevation, these tiles most probably contains features
+                // that lays directly on the gorund.
                 if (useElevationRangeSource) {
                     const range = elevationRangeSource!.getElevationRange(childTileKey);
                     geoBox.southWest.altitude = range.minElevation;
                     geoBox.northEast.altitude = range.maxElevation;
-
                     calculationFinal =
                         calculationFinal &&
                         range.calculationStatus === CalculationStatus.FinalPrecise;
                 }
 
                 let subTileArea = 0;
 
-                if (this.m_projection.type === ProjectionType.Spherical) {
+                const strictIntersections: boolean =
+                    this.m_projection.type === ProjectionType.Spherical;
+                // || useElevationRangeSource;
+                if (strictIntersections) {
```

Comment:
Not sure if strict is the proper name, b/c in planar projection the obb and the aabb are identical so they are both strict.

Reply:
Ok.